### PR TITLE
Hometask 3

### PR DESCRIPTION
**Pull request by bogdanbasii**
All is good except one thing: Python defines symbols as lowercases in my case. Is there any solution?

### CODE REVIEW

File: hometask3.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+for item in list:
+    if item.isdigit():
+        if int(item) % 2 == 0:
+            print(f"{item} is even")
+        else:
+            print(f"{item} is odd")
+    elif item.isupper():
+        print(f"{item} is a capital letter")
+    else:
```
Comment (klymenok) [inline hometask3.py]: You can add here another check:
```
elif item.islower() # for all lowercase letter
    ...
else: # for everything else (symbol)
    ...

File: hometask3.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+x = input("Enter a symbol:")
+
+list= list(x)
```
Comment (klymenok) [inline hometask3.py]: You don't need this transformation. You can iterate through string directly

File: hometask3.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        print(f"{item} is a lowercase letter")
+else:
+    print(f"{item} is a symbol")
+
+
+
+
+import time
+a = 0
+while a < 10:
```
Comment (klymenok) [inline hometask3.py]: To create an infinite loop you can simply use `while True`

**Review by klymenok — CHANGES_REQUESTED** [review]

File: hometask3.py (hunk 1 above)
Comment (bogdanbasii) [reply hometask3.py]: I've done the changes. Please take them into consideration

**Comment by klymenok** [comment]
Check my other comments, please

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**